lib/admin/components/UserSettings.js

```diff
@@ -1,20 +1,34 @@
 // @flow
 
-import React, {Component, type Node} from 'react'
-import {Row, Col, Checkbox, Panel, SplitButton, MenuItem, Label as BsLabel} from 'react-bootstrap'
+import React, { Component, type Node } from 'react'
+import {
+  Checkbox,
+  Col,
+  ControlLabel,
+  FormControl,
+  FormGroup,
+  Label as BsLabel,
+  MenuItem,
+  Panel,
+  Row,
+  SplitButton
+} from 'react-bootstrap'
 import update from 'react-addons-update'
 import Select from 'react-select'
 
 import UserPermissions from '../../common/user/UserPermissions'
-import {getComponentMessages} from '../../common/util/config'
+import { getComponentMessages } from '../../common/util/config'
+import { getDefaultAccountType } from '../../common/util/user'
 import * as feedActions from '../../manager/actions/feeds'
-import type {Permission, PermissionType} from '../../common/user/UserPermissions'
-import type {Organization, Project} from '../../types'
-import type {ManagerUserState} from '../../types/reducers'
+import type { Permission, PermissionType } from '../../common/user/UserPermissions'
+import type { AccountType, Organization, Project } from '../../types'
+import type { ManagerUserState } from '../../types/reducers'
 
 import ProjectAccessSettings from './ProjectAccessSettings'
 
 type Props = {
+  accountType?: ?string,
+  accountTypes: Array<AccountType>,
```

Comment:
Is there any way we can get this in from redux in this file instead of drilling it all the way in?

Reply:
Unfortunately this is not possible without heavier work. `connect`-ing to redux causes existing code to lose access to public class methods.

Reply:
I made the account type selector a connected component in https://github.com/ibi-group/datatools-ui/pull/855/commits/406555e72d66df5c782f929bc11c9e61e50642f4.